ocd_backend/transformers/__init__.py

```diff
@@ -21,23 +20,10 @@ def deserialize_item(content_type, raw_item):
         else:
             raise NoDeserializerAvailable('Item with content_type %s' % content_type)
 
-    def start(self, content_type, raw_item, entity, source_item, **kwargs):
-        """Start transformation of a single item."""
+    # def __init__(self, source_definition):
+    #     self.source_definition = source_definition
 
-        self.source_definition = kwargs['source_definition']
-        item_class = load_object(self.source_definition['item'])
 
-        deserialized_item = self.deserialize_item(content_type=content_type, raw_item=raw_item)
-
-        transformed_item = item_class(deserialized_item=deserialized_item,
-                                      entity=entity,
-                                      source_definition=self.source_definition).transform()
-
-        transformed_item.save()
-
-        return transformed_item
-
-
-@celery_app.task(bind=True, base=BaseTransformer, autoretry_for=(Exception,), retry_backoff=True)
-def transformer(self, *args, **kwargs):
-    return self.start(*args, **kwargs)
+from goapi_committee import *
```

Comment:
This is necessary to register the Tasks with Celery.

Reply:
This is not a best practice, I think that `ocd_backend.transformers.*` in `ocd_backend/__init__.py` would work better.

Reply:
Wildcard import doesn't work there. The only alternative is to register all tasks by name individually.

Reply:
You are correct, I forgot that wildcards did not work there.

I think I have found a solution, see https://docs.celeryproject.org/en/latest/userguide/tasks.html#how-it-works:

> This is the list of tasks built into Celery. Note that tasks will only be registered when the module they’re defined in is imported.
> The default loader imports any modules listed in the imports setting.

So maybe we can use `imports` instead of `include` to avoid having to register them all individually.